.run/SDK build.run.xml

```diff
@@ -0,0 +1,15 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
I think these are IDE configuration files? Probably you should add this folder to the `.gitignore`